### ISSUE TITLE
Suggested "update frequency" field

### ISSUE THREAD

**Issue by everydayduffy** [3liz/qgis-pgmetadata-plugin#80]
Hello,

As a "GIS Administrator" using your plugin (thanks for making it), it would be useful to have an "update frequency" field. This would help keep track of datasets which require regular updating or identify those that have been produced as a one-off. On many occasions this differs from publication frequency, which is often determined by a 3rd party. 

Perhaps it could go under the "Data last update" field? 

![image](https://user-images.githubusercontent.com/15983662/132705625-9183e1c5-492d-4aea-901c-918a840c4906.png)

**Comment by effjot** [comment]
If I understand correctly, you consider this field for “in-house” updates, i.e. for your own data which you need to update regularly (or mark as “one-off” / “doesn’t have to be updated”), so you can see for which datasets updates have to be scheduled?

**Comment by everydayduffy** [comment]
Yes - that's exactly it.

**Comment by effjot** [comment]
This doesn’t seem to be difficult to implement. However, I don’t know what @Gustry thinks about it – it is another data field for users to understand and for the developers to maintain.

What values would you put in there? Similar to publication frequency (yearly, monthly, etc.) and something like “no updates necessary” (for one-off datasets)?

**Comment by mdouchin** [comment]
You mean something like the field `publication_frequency` with the same glossary ?

`| publication_frequency | text | 2147483647 | √ |   | null |   |   | Frequency of publication: how often the data is published |`

Present glossary for this field is 
```
'YEA', 'Yearly', 'Update data yearly'
'MON', 'Monthly', 'Update data monthly'
'WEE', 'Weekly', 'Update data weekly'
'DAY', 'Daily', 'Update data daily'
```

**Comment by everydayduffy** [comment]
Yes a very similar or identical glossary would be useful should this new field be implemented. @effjot - a "no updates required" option would also make sense. 

From a practical perspective, if I were to have a field such as this in place, I could then query it alongside "Data last update" to see which datasets require updates. For example if update frequency = monthly, but the data last update = 3 months ago, I could flag that that particular dataset is "out of date".
